tests/e2e/utils/src/flows/with-rest-api.js

```diff
@@ -93,6 +93,80 @@ export const withRestApi = {
 		const repository = Order.restRepository( client );
 		await deleteAllRepositoryObjects( repository, null, orderStatuses );
 	},
+	/**
+	 * Adds a shipping zone along with a shipping method using the API.
+	 *
+	 * @param zoneName Shipping zone name.
+	 * @param zoneLocation Shiping zone location. Defaults to country:US. For states use: state:US:CA.
+	 * @param zipCode Shipping zone zip code. Default is no zip code.
+	 * @param zoneMethod Shipping method type. Defaults to flat_rate (use also: free_shipping or local_pickup).
+	 * @param cost Shipping method cost. Default is no cost.
+	 * @param additionalZoneMethods Array of additional zone methods to add to the shipping zone.
+	 */
+	addShippingZoneAndMethod: async (
+		zoneName,
+		zoneLocation = 'country:US',
+		zipCode = '',
+		zoneMethod = 'flat_rate',
+		cost = '',
+		additionalZoneMethods = [] ) => {
+
+	   const path = 'wc/v3/shipping/zones';
+
+	   const response = await client.post( path, { name: zoneName } );
+	   expect(response.statusCode).toEqual(201);
+	   let zoneId = response.data.id;
+
+	   // Select shipping zone location
+	   let zoneType = zoneLocation.split(/:(.)/)[0];
+	   let zoneCode = zoneLocation.split(/:(.+)/)[1];
+	   let zoneLocationPayload = [
+		   {
+			   code: zoneCode,
+			   type: zoneType
+		   }
+	   ];
+
+	   // Fill shipping zone postcode if provided
+	   if ( zipCode ) {
+		   zoneLocationPayload.push( {
+			   code: zipCode,
+			   type: "postcode",
+		   } );
+	   };
```

Comment:
Extraneous comma.

Reply:
Removed 👍

Reply:
@zhongruige Sorry, I meant to say extraneous semi-colon on the end of the `if` block. The commas in the object declarations were okay.

Reply:
No worries! Updated :D